app/build.gradle

```diff
@@ -40,7 +40,7 @@ android {
     }
     buildTypes {
         release {
-            minifyEnabled true
+            minifyEnabled false
```

Comment:
Does minify break now? I'd like the project to keep minify functional so that users who need minify don't all have to figure out the rules needed for Reark